src/ServiceManagement/Common/Commands.ScenarioTest/Resources/ServiceManagement/ServiceManagementTests.ps1

```diff
@@ -912,6 +912,30 @@ function Test-MigrationAzureReservedIP
     Assert-AreEqual $removeReservedIP.OperationStatus "Succeeded"
 }
 
+<#
+.SYNOPSIS
+Tests New-AzureReservedIPWithTags
+#>
+function Test-AzureReservedIPWithIPTags
+{
+    # Setu
+    $name = getAssetName
+    $location = "West Central US"
```

Comment:
Yes, the issue is not the region you are using,. it is that you are using a hard-coded region.  You should get the region by querying for the available regions for this resource.  Without this, it is not possible to run this test against other azure instances (test cluster, sovereign clouds, etc.)